app/presenters/case_study_presenter.rb

```diff
@@ -2,10 +2,7 @@ class CaseStudyPresenter < ContentItemPresenter
   include ContentItem::Body
   include ContentItem::Metadata
   include ContentItem::TitleAndContext
-
-  def image
-    content_item["details"]["image"]
-  end
+  include ContentItem::NewsImage
```

Comment:
Should this be in case study? I thought we didn't support that type in rummager so probably should be the same here

Reply:
You're absolutely right, now removed.